src/components/ProjectRequestForm/FormForProfit.jsx

```diff
@@ -0,0 +1,157 @@
+import React from "react";
+import styles from "./requestForm.module.scss";
+import { Formik, Form, Field, ErrorMessage } from "formik";
+import * as Yup from "yup";
```

Comment:
Is Yup a component? If not. it should start with lower case.

Reply:
I saw few ways this is being done online - changed to lowercase.